README.md

```diff
@@ -50,3 +50,6 @@ For more information, see the [Hugo theme customization documentation](https://g
 
 [Creative Commons Attribution](https://creativecommons.org/licenses/by/4.0/) for [content](content) / [MIT License](https://github.com/inbo/tutorials/blob/master/LICENSE) for source code.
 
+
+
+TODO: update docs about authors
```

Comment:
I'm not sure whether this change in the Readme is still needed? Is this fulfilled by your update of the 'create tutorial' tutorial? If so, it can be dropped (else, not clear what exactly is meant).